tests/functional/test_init.py

```diff
@@ -37,3 +39,11 @@ def test_version_option():
     assert not ret.stderr, "Nothing should be written to stderr"
     assert ret.returncode == 0, "A non-successful return code was provided"
     assert ret.stdout == expected_output, "Output did not match expected input"
+
+
+def test_phylum_pubkey_is_constant(tmp_path):
```

Comment:
Do you think it might make sense to verify at runtime that the public key hosted on github and the public key embedded in phylum-init is the same?

I don't think it's necessary since we always use the internally embedded key anyway so there's no risk of the remote one being swapped (and the embedded one being swapped would mean arbitrary code execution from phylum-init anyway), just curious if you've considered it.

Reply:
It was considered and decided against since making network calls to GitHub has been found in the past to be a source of potential failure. The addition of a functional test to compare the embedded key to the network hosted one was made as a means to identify when the keys don't match. That also has the advantage of "spreading" the public key to multiple locations so that a change to it (malicious or benign) will require access and coordination to multiple sources. It is understood that this method is not fool proof but should help to identify failures to the Phylum devs sooner.